### PR TITLE
Bump css-loader version from 4.3.0 to 6.11.0

### PR DESCRIPTION
**Pull request by ayusht2810**
#### Summary
- Bump the css-loader version from 4.3.0 to 6.11.0
- Updated webpack version and webpack config file

#### Ticket Link
https://github.com/mattermost/mattermost-plugin-gitlab/security/dependabot/32

#### Checklist
- [x] Completed dev testing
- [x] `make test` Ran test cases and ensured they are passing
- [x] `make check-style` Ran style check and ensured both webapp and server pass the checks

### CODE REVIEW

**Comment by mickmister** [comment]
I see some linting failing due to some things in an external library `node-polyfill-webpack-plugin`. Maybe we should change the following to `./src/*`

https://github.com/mattermost/mattermost-plugin-gitlab/blob/191a11a9b91c308d9613f9d13a12dbe330de6b2d/webapp/tsconfig.json#L26

---

https://github.com/mattermost/mattermost-plugin-gitlab/actions/runs/8784768277/job/24103695559?pr=492

```
> gitlab@0.1.0 lint
> eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx --ext tsx --ext ts . --quiet --cache

cd webapp && npm run check-types

> gitlab@0.1.0 check-types
> tsc

Error: node_modules/node-polyfill-webpack-plugin/node_modules/type-fest/source/array-slice.d.ts(88,53): error TS1005: '?' expected.
Error: node_modules/node-polyfill-webpack-plugin/node_modules/type-fest/source/array-slice.d.ts(88,61): error TS1005: ':' expected.
Error: node_modules/node-polyfill-webpack-plugin/node_modules/type-fest/source/array-slice.d.ts(89,4): error TS1005: ':' expected.
Error: node_modules/node-polyfill-webpack-plugin/node_modules/type-fest/source/array-slice.d.ts(92,4): error TS1005: ',' expected.

```

**Comment by mickmister** [comment]
@ayusht2810 There are merge conflicts from merging a different PR

File: webapp/webpack.config.js [hunk 1]
```diff
@@ -47,6 +49,9 @@ module.exports = {
         'prop-types': 'PropTypes',
         'react-bootstrap': 'ReactBootstrap',
     },
+    plugins: [
+        new NodePolyfillPlugin(),
+    ],
```
Comment (mickmister) [inline webapp/webpack.config.js:54]: What's the main reason for this. Were we getting a `process` not defined error?

Comment (ayusht2810) [reply webapp/webpack.config.js:54]: This was done due to the upgrade of webpack version to `5, You can read more about it here: https://webpack.js.org/migrate/5/#upgrade-webpack-to-5, https://webpack.js.org/migrate/5/#upgrade-webpack-to-5, https://stackoverflow.com/a/65556946

Comment (mickmister) [reply webapp/webpack.config.js:54]: What error do we get without it though?

Comment (mickmister) [reply webapp/webpack.config.js:54]: It's fine but this means we should add it to other plugin projects that will run into this issue too. I wonder if we even need this though. Not sure what Nodejs functions we would need

**Review by mickmister — APPROVED** [review]

File: webapp/webpack.config.js (hunk 1 above)
Comment (mickmister) [reply webapp/webpack.config.js:54]: @ayusht2810 What error do we get without adding this?

Comment (ayusht2810) [reply webapp/webpack.config.js]: @mickmister Not sure why we are not getting error now. I have reverted the changes and requested QA review again.

**Comment by ayusht2810** [comment]
@AayushChaudhary0001 Can you please test the PR again?

**Comment by AayushChaudhary0001** [comment]
@ayusht2810 I have re-tested this PR again, it is working fine, LGTM. Approved

**Pull request merged**